os/services/orchestra/orchestra-rule-unicast-per-neighbor-rpl-ns.c

```diff
@@ -104,7 +104,7 @@ init(uint16_t sf_handle)
   /* Add a Tx link at each available timeslot. Make the link Rx at our own timeslot. */
   for(i = 0; i < ORCHESTRA_UNICAST_PERIOD; i++) {
     tsch_schedule_add_link(sf_unicast,
-        LINK_OPTION_SHARED | LINK_OPTION_TX | ( i == rx_timeslot ? LINK_OPTION_RX : 0 ),
+        LINK_OPTION_SHARED | LINK_OPTION_TX | ( i == rx_timeslot ? LINK_OPTION_RX : 0 ) ,
```

Comment:
This is probably an accidental change. Same in `orchestra-rule-unicast-per-neighbor-rpl-storing.c`